quantum/process_keycode/process_combo.c

```diff
@@ -175,3 +193,26 @@ void matrix_scan_combo(void) {
     dump_key_buffer(true);
   }
 }
+
+void combo_enable(void) {
+    is_combo_enable = true;
+}
+
+void combo_disable(void) {
+    is_combo_enable = is_active = false;
+    timer = 0;
+    dump_key_buffer(true);
+
+}
+
+void combo_toggle(void) {
+    if (is_combo_enable) {
+        combo_disable();
+    } else {
+        combo_enable();
+    }
+}
+
+bool get_combo_enable(void) {
```

Comment:
Maybe change this to `is_combo_enabled` so it reads slightly better?

Reply:
I had considered that initially, but I don't like `combo_enabled` for the variable name.  That seems problematic/confusing to me.

Reply:
Fair enough, coming from Java there is something of a rule to use `is` for boolean getters, but I couldn't find anything specifically about C. Probably because it's not OO 😄

Reply:
Well, I'm horrible with naming... so if somebody has a better suggestion, I'm totally open to it.

Reply:
I definitely agree that `is_combo_enabled` reads better here.

Not sure yet about the variable name.

Reply:
I updated this anyways.  though, I'm not sure if `b_combo_enabled` is a good name for the variable.

Reply:
I would say `b_` isn't sympathetic to the surrounding code, *shrug* maybe it's the next code style to tackle like the recent PP stuff.